datastore/tasks/tasks.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by the Apache 2.0
 // license that can be found in the LICENSE file.
 
-// [START all]
+// [START datastore_all]
```

Comment:
Is this the canonical name of this region?

Reply:
I'm not sure if this is used at all, but am fixing it now incase it is used in the future.